cog/daily_charge.py

```
@@ -23,95 +25,293 @@
-            # 紀錄log
-            # pylint: disable-next = line-too-long
-            print(
-                f"{interaction.user.id},{interaction.user} Get 5 point by daily_charge {datetime.now()}"
+            return
+
+        with mysql_connection() as c:  # SQL 會話
+            _, cursor = c
+            user = interaction.user
+
+            # get now time and combo
+            now = datetime.now().replace(microsecond=0)
```

Ruff (0.11.9)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)